docs/examples/driver_examples/Qcodes example with keithley 2450.ipynb

```diff
@@ -18,8 +18,12 @@
     "from visa import VisaIOError\n",
```

Comment:
Not related here but this should be pyvisa and not the long deprecated visa module